<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>828</width>
    <height>695</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QListWidget" name="list_order">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>701</width>
     <height>231</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_up">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>60</y>
     <width>91</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Up</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_down">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>90</y>
     <width>91</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Down</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_refresh">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>390</y>
     <width>91</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Refresh</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_reque">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>210</y>
     <width>91</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Return to Que</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_top">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>30</y>
     <width>91</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Top</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_bottom">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>120</y>
     <width>91</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Bottom</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_open_folder">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>240</y>
     <width>91</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Open Folder</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="chk_show_completed">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>280</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Show Completed</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="tristate">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_kill_advance">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>270</y>
     <width>91</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Kill / Advance</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_kill_stop">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>300</y>
     <width>91</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Kill / Stop Que</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_stop">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>330</y>
     <width>91</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Stop Que</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_start">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>360</y>
     <width>91</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Start Que</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="text_job_info">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>310</y>
     <width>701</width>
     <height>131</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_job_list">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>201</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>Job Que</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_que_log">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>450</y>
     <width>391</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Que Log</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>10</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Job Commands</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="text_log">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>470</y>
     <width>801</width>
     <height>221</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_job_info">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>290</y>
     <width>46</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Job Info</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_mark_single">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>150</y>
     <width>91</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Mark Single</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_mark_full">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>180</y>
     <width>91</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Mark Full</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_job_list_columns">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>701</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Consolas</family>
    </font>
   </property>
   <property name="text">
    <string>Job Path                                                              | Run Type           | Status</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
